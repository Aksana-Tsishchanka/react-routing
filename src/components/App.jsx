import React from 'react';
import { Link } from 'react-router';

export default class App extends React.Component {
   render() {
      return (
          <div>
            <ul>
               <li><Link to="/login">Login</Link></li>
               <li><Link to="/todo">TO DO list</Link></li>
            </ul>   
           {this.props.children}
         </div>
      );
   }
}

class Login extends React.Component {
   render() {
      return (
         <div>
            <h1>Login page...</h1>
         </div>
      )
   }
}

class TodoList extends React.Component {
   render() {
      return (
         <div>
            <h1>TO DO page...</h1>
         </div>
      )
   }
}