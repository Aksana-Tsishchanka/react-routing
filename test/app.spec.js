import React from 'react';
import renderer from 'react-test-renderer';
import App from '../src/components/App';


test('App render', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
