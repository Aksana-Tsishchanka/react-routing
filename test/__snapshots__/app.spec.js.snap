// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render 1`] = `
<div>
  <a
    onClick={[Function]}
    style={Object {}}
  >
    Log In
  </a>
</div>
`;
